src/components/TopBar/TopBar.scss

```
@@ -128,7 +129,8 @@ $context-control-expand-after: 1400px;
 }
 
 .SecondaryMenu {
-  @include breakpoint-before(layout-width(page-with-nav), false) {
-    display: none;
+  svg {
+    color: transparent;
+    fill: var(--p-icon, var(--top-bar-color));
   }
```

We could use recolor-icon for this
```suggestion
  @include recolor-icon(var(--p-icon, var(--top-bar-color)), transparent);
```